README.md

```diff
@@ -69,6 +70,18 @@ This work is licensed under a [Creative Commons Attribution-ShareAlike 2.0 Gener
 - Never assume you don’t need a CoC because nobody reported any issues, and therefore you don’t need one. As Ola notes, it takes time for people to feel comfortable to report issues. And not having a CoC at all is sending a very strong message to everyone that reporting isn’t worthwhile. It doesn't mean people won't experience issues - it just means they won't tell you. They're likely to tell their friends though.
 - For more about CoC enforcement, see this [overview of CoC warning signs](https://otter.technology/blog/2017/12/28/code-of-conduct-enforcement-warning-signs/) by Sage Sharp.
 
+## Footprint
```

Comment:
It's a really good idea to give an idea of where the impact normally is, so organiser's actions are in proportion to the having some impact on the environmental footprint of an event. Wikimania share some figures of an event to give context. you can see how flying is by far the largest impact, at more than 80% of the impact:

https://wikimania.wikimedia.org/wiki/2019:Carbon_offsetting

Reply:
Thanks so much, that's super useful, have added this below. Maybe you have more in terms of background.

I think it makes sense to add a little bit of background, because Environmental issues are exceptionally prone to technical scrutiny, and of course it makes sense that a conference doesn't just get bio-degradable cups and then pretend to be green.. some pointers to more background information seems like a good idea, and maybe still "Less Obvious" as the title of this doc outlines :)